src/components/layout/AddButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, Grid, Icon } from '@mui/material';
+import React from 'react';
+
+function AddButton({ onClick }: any) {
```

Comment:
Make it a reusable component. it should be a button component. You have named it by the action it's performing. If your application expand this component can not be used.